main/views.py

```diff
@@ -1,3 +1,11 @@
 from django.shortcuts import render
+from .models import Food
 
-# Create your views here.
+def index(request):
+    foods = Food.objects.order_by('-rating')
```

Comment:
```suggestion
    foods = Food.objects.select_related('images').order_by('-rating')
```

Reply:
![image](https://user-images.githubusercontent.com/50240885/129655848-93d8999f-8be9-4f5c-b27e-9d764d74fc31.png)

Chị ơi hình như `select_related()` chỉ dùng được với quan hệ OneToOne với những field là Foreign key của bảng đó thôi ạ, ví dụ nếu images là foreign key của Food thì mới dùng được ạ :/

Reply:
em dùng `prefetch_related` cho many-to-one, many-to-many nha

Reply:
Dạ vâng em sửa rồi ạ